backend/app/views/spree/admin/products/index.html.erb

```diff
@@ -78,7 +78,7 @@
               <%= render 'spree/admin/shared/image', image: product.gallery.images.first, size: :mini %>
             </td>
             <td><%= link_to product.try(:name), edit_admin_product_path(product) %></td>
-            <td class="align-right"><%= product.display_price.to_html %></td>
+            <td class="align-right"><%= product.display_price&.to_html %></td>
```

Comment:
Why this is needed?

Reply:
In fact, it's not needed. Probably I used it as an intermediate step, and it leaked to the final result. Thanks for catching it! I forced a push and removed the change.

Reply:
I can still see this change, maybe some mess with git?